var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -44,16 +45,28 @@ class Sensei(CMakePackage):
     # All SENSEI versions up to 2.1.1 support only Python 2, so in this case
     # Paraview 6 cannot be used since it requires Python 3. Starting from
     # version 3, SENSEI supports Python 3.
-    depends_on("paraview@5.5.0:5.5.2+mpi+hdf5", when="@:2.1.1 +catalyst")
-    depends_on("paraview@5.5.0:5.5.2+python+mpi+hdf5", when="@:2.1.1 +catalyst+python")
-    depends_on("paraview@5.6:5.7+mpi+hdf5", when="@3:3.2.1 +catalyst")
-    depends_on("paraview@5.6:5.7+python3+mpi+hdf5", when="@3:3.2.1 +catalyst+python")
-    depends_on("paraview+mpi+hdf5", when="+catalyst")
-    depends_on("paraview+python3+mpi+hdf5", when="+catalyst+python")
+    depends_on("paraview+mpi", when="+catalyst")
+    depends_on("paraview+hdf5", when="+catalyst+hdf5")
+    depends_on("paraview+python", when="@:2 +catalyst+python")
+    depends_on("paraview+python3", when="@3: +catalyst+python")
+
+    depends_on("paraview@5.5.0:5.5.2", when="@:2.1.1 +catalyst")
+    depends_on("paraview@5.6:5.7", when="@3:3.2.1 +catalyst")
+    depends_on("paraview@5.7:5.9", when="@3.2.2 +catalyst")
+    depends_on("paraview@5.7:5.10", when="@4: +catalyst")
+
+    # Visit Dep
     depends_on("visit~gui~python", when="+libsim")
-    depends_on("vtk@8.1.0:8.1.2", when="+libsim")
```

Comment:
Removed this, visit is responsible for constraining the version of VTK it can use, sensei will work with what it is given.